src/diode.js

```diff
@@ -18,15 +25,26 @@ function Diode(target) {
    * Callbacks are eventually executed, Diode does not promise
    * immediate consistency so that state propagation can be batched
    */
-  var _flush = function() {
+  var _flush = function(args) {
     /**
      * Important: do not cache the length of _callbacks
      * in the event a callback causes later subscriptions
      * to disappear
      */
     for (var i = 0; i < _callbacks.length; i++) {
-      _callbacks[i].apply(target, arguments)
+      _callbacks[i].apply(target, args)
+    }
+  }
+
+  var _cancel = function() {
+    var now = +new Date() // IE8 love
+
+    if (_lastFire && now - _lastFire < 10) {
+      cancelAnimationFrame(_tick)
```

Comment:
Why do you need to cancel? Why can't the _tick method just run whatever the most current information is? Which could be nothing I guess. Something like:

```
var currentCallbacks = []
// Update ^ however frequently you need to outside of requestAnimationFrame

requestAnimationFrame(function(){
  currentCallbacks.forEach((callback) => callback())
})
```